sdk/src/events.ts

```diff
@@ -44,22 +44,66 @@ export async function findEventTopicLog<T>(
   }
   return iface.parseLog(log) as LogDescription & (T | LogDescription);
 }
+
+export const CALLBACK_HANDLER_EVENTS = {
+  CallbackReceived: 'CallbackReceived',
+};
+
+export const DAO_EVENTS = {
+  NewURI: 'NewURI',
+};
+
+export const DAO_REGISTRY_EVENTS = {
+  DAORegistered: 'DAORegistered',
+};
+
 export const IPROPOSAL_EVENTS = {
-  PROPOSAL_CREATED: 'ProposalCreated',
-  PROPOSAL_EXECUTED: 'ProposalExecuted',
+  ProposalCreated: 'ProposalCreated',
+  ProposalExecuted: 'ProposalExecuted',
 };
 
 export const IDAO_EVENTS = {
-  METADATA_SET: 'MetadataSet',
-  EXECUTED: 'Executed',
-  DEPOSITED: 'Deposited',
-  STANDARD_CALLBACK_REGISTERED: 'StandardCallbackRegistered',
-  TRUSTED_FORWARDER_SET: 'TrustedForwarderSet',
-  NEW_URI: 'NewURI',
+  MetadataSet: 'MetadataSet',
```

Comment:
Didnt notice till now, but why are those exported as objects and not enums

Reply:
There are somehow problems with `const enums` https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-runtime that I cannot oversee atm.
Moreover, my assignment wouldn't work anymore
![Bildschirmfoto 2024-01-24 um 16 08 18](https://github.com/aragon/osx-commons/assets/20623991/ce7adec8-a2e5-4047-846a-3b065747715e)


I will leave it as it is.